book/10-git-internals/sections/transfer-protocols.asc

```diff
@@ -153,9 +153,9 @@ $ ssh -x git@server "git-receive-pack 'simplegit-progit.git'"
 The `git-receive-pack` command immediately responds with one line for each reference it currently has – in this case, just the `master` branch and its SHA-1.
 The first line also has a list of the server's capabilities (here, `report-status`, `delete-refs`, and some others, including the client identifier).
 
-Each line starts with a 4-character hex value specifying how long the rest of the line is.
-Your first line starts with 00a5, which is hexadecimal for 165, meaning that 165 bytes remain on that line.
-The next line is 0000, meaning the server is done with its references listing.
+The data is transmitted in chunks. Each chunk starts with a 4-character hex value specifying how long the chunk is (including the 4 bytes of the length itself). Chunks usually contain a single line of data and a trailing linefeed.
```

Comment:
Could we stick to the one-sentence-per-source-line convention here? It makes future diffs much easier to read.